proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,393 @@
+  Type type = 3;
+  // Topics is the list of topics that are being mounted or unmounted.
+  repeated Topic topics = 4;
+}
+
+message ListMountTasksRequest {}
+
+message ListMountTasksResponse {
+  repeated MountTask tasks = 1;
+}
+
+message GetMountTaskRequest {
+  // ID is the unique identifier of the mount task to retrieve.
+  int32 id = 1 [
+    // We won't add validation required here, because 0 may be a valid id.
+    (google.api.field_behavior) = REQUIRED
+  ];
+}
+
+message GetMountTaskResponse {
+  MountTask task = 1;
+}
+
+message DeleteMountTaskRequest {
+  // ID is the unique identifier of the mount task to delete.
+  int32 id = 1 [(google.api.field_behavior) = REQUIRED /* We won't add validation required here, because 0 may be a valid id. */];
+}
+
+message DeleteMountTaskResponse {}
+
+message UpdateMountTaskRequest {
+  enum Action {
+    ACTION_UNSPECIFIED = 0;
+    ACTION_PREPARE = 1;
+    ACTION_EXECUTE = 2;
+    ACTION_FINISH = 3;
+    ACTION_CANCEL = 4;
+  }
+
+  // ID is the unique identifier of the mount task to delete.
+  int32 id = 1 [(google.api.field_behavior) = REQUIRED /* We won't add validation required here, because 0 may be a valid id. */];
+
+  // Action is the action to execute on mount task.
+  Action action = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true,
+    (buf.validate.field).enum = {
+      defined_only: true
+      not_in: [0] /* Disallow: Unspecified */
+    }
+  ];
+}
+
+message UpdateMountTaskResponse {}
+
+service CloudStorageService {
+  rpc MountTopics(MountTopicsRequest) returns (MountTopicsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/topics/mount"
+      body: "topics"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Mount topics from tiered storage"
+      description: "This operation restores topics that were offloaded to tiered storage, making them available for consumption and production again. Remounting a topic reloads its data and state to the local brokers, allowing active use of the topic while consuming some cluster resources."
+      responses: {
+        key: "200"
+        value: {
+          description: "OK"
+          schema: {
+            json_schema: {ref: ".redpanda.api.dataplane.v1alpha2.MountTopicsResponse"}
+          }
+        }
+      }
+      responses: {
+        key: "404"
+        value: {
+          description: "Not Found"
+          schema: {
+            json_schema: {ref: ".google.rpc.Status"}
+          }
+        }
+      }
+    };
+  }
+
+  rpc UnmountTopics(UnmountTopicsRequest) returns (UnmountTopicsResponse) {
+    option (google.api.http) = {
+      post: "/v1alpha2/cloud-storage/topics/unmount"
+      body: "topics"
+      response_body: "*"
+    };
+    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "Unmount topics to tiered storage"
```

Comment:
```suggestion
      summary: "Unmount topics to object storage"
```
What do you think of standardizing to "object storage"? In the [doc]((https://deploy-preview-725--redpanda-docs-preview.netlify.app/24.3/manage/mountable-topics/) we do specify that Tiered Storage is a prereq, but then we say object storage when referring to topics that were unmounted to the cloud.